mods/doors/init.lua

```diff
@@ -460,7 +468,7 @@ doors.register("door_wood", {
 })
 
 doors.register("door_steel", {
-		tiles = {{name = "doors_door_steel.png", backface_culling = true}},
+		tiles = {{ name = "doors_door_steel.png", backface_culling = true }},
```

Comment:
There shouldn't be spaces just inside the brackets.

Reply:
Oh OK, I did it to match the code in the other door. Shall I remove that one too?

Reply:
Thanks, resolved.